light-node/src/commands/initialize.rs

```diff
@@ -0,0 +1,110 @@
+//! `intialize` subcommand
+
+use crate::application::app_config;
+use crate::config::LightClientConfig;
+
+use std::collections::HashMap;
+
+use abscissa_core::status_err;
+use abscissa_core::status_warn;
+use abscissa_core::Command;
+use abscissa_core::Options;
+use abscissa_core::Runnable;
+
+use tendermint::hash;
+use tendermint::lite::Header;
+use tendermint::Hash;
+
+use tendermint_light_client::components::io::{AtHeight, Io, ProdIo};
+use tendermint_light_client::operations::ProdHasher;
+use tendermint_light_client::predicates::{ProdPredicates, VerificationPredicates};
+use tendermint_light_client::store::sled::SledStore;
+use tendermint_light_client::store::LightStore;
+use tendermint_light_client::types::Status;
+
+/// `initialize` subcommand
+#[derive(Command, Debug, Default, Options)]
+pub struct InitCmd {
+    #[options(
+        free,
```

Comment:
What's the rationale for having the `height` and `hash` as free options rather than names ones? Is that because those are required options, and we'd rather leave named options for optional parameters?

While it's unlikely/almost impossible one would switch the two options around since the hash could never be parsed as a u64, I would personally rather have them named explicitly, although I don't feel strongly either way.

Reply:
> Is that because those are required options, and we'd rather leave named options for optional parameters?

Yes exactly. Seems like in absicca or rather gumdrop this seems the way to say these are mandatory. I think we can remove the `free` make them named and manually check if they were provided. Let's move this to a followup issue.